README.md

```diff
@@ -8,29 +8,27 @@ See [Downloading and Opening](./INSTALLING.md) for instructions. All you need is
 
 ## License and giving credit
 
-All content in this repository is under a [Creative Commons Attribution-ShareAlike 2.0 license](./LICENSE) unless otherwise stated.
-
-This means anyone is allowed to use, adapt, and reshare everything I made in here, under the following conditions:
-
-- You give attribution (preferably a mention of my name, a link to this repository, and the copyright license in use). It's best to provide attribution both in the repository containing your Scratch project (if you have one) and inside your Scratch project itself.
-  <details open><summary>For example:</summary>
-
-  <p>
-  
-  > #### In your source code repository
-  > Credits:
-  > - [Video Player Maker](https://github.com/DNin01/Scratch-projects/tree/main/turbowarp/tools/video-player) by [@DNin01](https://github.com/DNin01) (CC-BY-SA 2.0)
-  > #### Inside your Scratch project
-  > Credits:
-  > - Video Player Maker by DNin01
-  > 
-  > (links in source code)
-  
-  </p>
-
-  </details>
-- License your adaptations of the content under the same license.
-
-Provided those conditions are met, you can go ahead and use these in your own projects!
+Content in this repository is licensed under a [Creative Commons Attribution 4.0 license](./LICENSE) _unless otherwise stated_.
+
+You are free to use, adapt, and reshare that content, provided that you give attribution (preferably a mention of my name, a link to this repository, and the copyright license in use). It's best to provide attribution both in the repository containing your Scratch project file (if you have one) and inside your Scratch project itself.
+<details open><summary>For example:</summary>
+
+<p>
+
+> #### In your source code repository
+> Credits:
+> - [Video Player Maker](https://github.com/DNin01/Scratch-projects/tree/main/turbowarp/tools/video-player) by [@DNin01](https://github.com/DNin01) (CC-BY 4.0)
+> #### Inside your Scratch project
+> Credits:
+> - Video Player Maker by DNin01 (CC-BY 4.0)
+> 
+> (links in source code)
+
+</p>
+
+</details>
+
+> [!NOTE]
+> Up until commit [`2a7ad24`](https://github.com/DNin01/Scratch-projects/commit/2a7ad24ea42b27c086543a39e285d13bc9f187e2), the contents of this repository were licensed under a [Creative Commons Attribution-ShareAlike 2.0 license](https://creativecommons.org/licenses/by-sa/2.0/legalcode). For those already reusing content, you have the right to use newer revisions of the content under the new license.
```

Comment:
Hmm, so this revision is the last one that used CC-BY-SA 2.0 or it was the one that updated it to CC-BY 4.0? 👀